Sources/ApolloWebSocket/ApolloWebSocket.swift

```diff
@@ -14,6 +14,9 @@ public protocol ApolloWebSocketClient: WebSocketClient {
   
   /// The URLRequest used on connection. 
   var request: URLRequest { get set }
+  
+  /// Queue where the callbacks are executed
+  var callbackQueue: DispatchQueue { get set }
```

Comment:
In other places like in the `HTTPNetworkTransport`, we've allowed the callback queue to be specified at the point where a method like `query` is called so that different information can be processed on different callback queues. 

I'm not sure how straightforward it would be to do that with `subscription` since I haven't had a chance to dive into the real nitty gritty of this code, but what do you think about that architecture vs setting the callback queue per websocket instance?

Reply:
I was mimicking the implementation of the underlying Starscream `WebSocket`. This is the queue from which a websocket calls its `WebSocketDelegate` methods. As for the queue where the subscription callbacks (`Result<...> -> Void`) are processed, they could be added to the subscriber list to be used when notifying subscribers. I feel like this is beyond the scope of this PR.

I added this to be able to easily set the Starscream callback queue. It didn't see like a big deal since the `ApolloWebSocketClient` implementation is only ever changed during testing. (AFAIK)

Reply:
Ahhh right I forgot about that implementation detail. You're right, this makes more sense!